.changeset/pre.json

```diff
@@ -2,8 +2,8 @@
   "mode": "pre",
   "tag": "alpha",
   "initialVersions": {
-    "@temp-nui/nui": "0.0.0",
-    "@temp-nui/storybook-utils": "0.0.0"
+    "@sit-onyx/components": "0.0.0",
```

Comment:
I am not quite sure about this package name.
What do you think about just naming it `sit-onyx`?

I think its common practice to name by main package by the scope itself and use the scope only for additional packages. E.g. like `vue`, `prettier`, `storybook` etc.